components/crud-web-apps/tensorboards/backend/app/utils.py

```diff
@@ -17,19 +16,12 @@ def parse_tensorboard(tensorboard):
         "name": tensorboard["metadata"]["name"],
         "namespace": tensorboard["metadata"]["namespace"],
         "logspath": tensorboard["spec"]["logspath"],
-        "age": helpers.get_age(tensorboard),
+        "age": tensorboard["metadata"]["creationTimestamp"],
         "status": status.create_status(phase, message, "")
     }
 
     return parsed_tensorboard
 
-def getPVCName(pvc):
```

Comment:
Could you split the changes of this commit in 2?
1. date component related changes
2. PVC name related changes

When this PR is squashed & merged we will still maintain a description of this change in the final commit message

Reply:
makes sense

Reply:
force pushed the changed commits. Resolving this one